### ISSUE TITLE
[Effect Suggestion] [Meta] Again!

### ISSUE THREAD

**Issue by TheRealAyCe** [gta-chaos-mod/ChaosModV#520]
Choose a one-time effect and invoke that effect alongside the regularly chosen effect for the next N minutes.

If the effect is "Spawn Car", spawns a car every 30 seconds at the same time the normal effect triggers.

Maybe the one-time effect is determined by the first voting after the metaeffect starts? (only allow one-time-effect voting options)

**Comment by Da-Best** [comment]
As i understand you want to make a meta where the 1 effect is chosen and it keeps getting added on when the normal effect starts? If that is the case that has softlock potential as if you get obliterate all nearby peds that will just get you into a death/fail loop or even crash the game.

**Comment by TheRealAyCe** [comment]
Yes, in the same way there is softlock potential in Full Accelleration and the like. It's not that bad, since those effects (as well as Metaeffects) end after a while. Stop and Stare for example can completely break missions, but you can always just restart the mission by dying.
